src/Common/Common.Api/Datasets/Table.cs

```diff
@@ -30,5 +30,21 @@ public class Table
                 Measures = table.Measures?.Select(m => (Measure)m)
             };
         }
+
+        public static implicit operator PowerBI.Api.V2.Models.Table(Table table)
+        {
+            if (table == null)
+            {
+                return null;
+            }
+
+            return new PowerBI.Api.V2.Models.Table
+            {
+                Name = table.Name,
+                Columns = table.Columns?.Select(c => (PowerBI.Api.V2.Models.Column)c).ToList(),
```

Comment:
Could use .Cast() instead

Reply:
Cast gave me an error saying cannot convert IEnumerable to IList. Could you guide me the right code? I simply did .Cast<PowerBI.Api.V2.Models.Column>

Reply:
You still have to do ToList() at the end to get IList. If it gives you issues just keep it as Select().